src/utils/transformChunk.ts

```diff
@@ -0,0 +1,67 @@
+import { decode } from 'sourcemap-codec';
+import error from './error';
+import { OutputOptions, RawSourceMap } from '../rollup/types';
+import Graph from '../Graph';
+import Chunk from '../Chunk';
+
+export default function transformChunk(
+	graph: Graph,
+	chunk: Chunk,
+	code: string,
+	sourcemapChain: RawSourceMap[],
+	options: OutputOptions
+) {
+	return graph.plugins.reduce((promise, plugin) => {
+		if (!plugin.transformBundle) return promise;
+
+		return promise.then(code => {
+			return Promise.resolve()
+				.then(() =>
+					(plugin.transformChunk || plugin.transformBundle).call(
```

Comment:
I must admit I could not get `transformChunk` to be called at all in my experiments. A test to take some inspiration from would be really helpful.